ios/FluentUI/Button/Button.swift

```diff
@@ -224,6 +224,19 @@ open class Button: UIButton {
         update()
     }
 
+    open override func didUpdateFocus(in context: UIFocusUpdateContext, with coordinator: UIFocusAnimationCoordinator) {
+        guard let window = window, style == .primaryFilled else {
+            return
+        }
+        let highlightedColor = UIColor(light: Colors.primaryTint10(for: window),
```

Comment:
wouldn't a better fix be utilizing updatebackgroundcolor() because dangerFill has similar problems and also one place to grab all the right colors?

Reply:
I like the idea. That approach would probably consist of create another property like `isFocused` that would be set here and used in the `updateBackgroundColor()` method as @harrieshin suggested.

Reply:
Since button already has isFocused, I didn't add a new redundant hasFocus, but it looks like it does get updated before the call to didUpdateFocus (which makes sense), so we're able to use that in updateBackgroundColor.